components/builder-depot-client/src/lib.rs

```diff
@@ -725,15 +722,13 @@ impl Client {
     ///
     /// * Package cannot be found
     /// * Remote Builder is not available
-    pub fn show_package<I>(
+    pub fn show_package(
         &self,
-        package: &I,
+        package: &PackageIdent,
         channel: Option<&str>,
         token: Option<&str>,
         target: Option<&str>,
-    ) -> Result<Package>
-    where
-        I: Identifiable,
+    ) -> Result<PackageIdent>
```

Comment:
Is the return type change intentional here? If so, the documentation should be changed, too.

Reply:
Yes it is 100% intentional. I'll correct the documentation.

Reply:
We found that there were only 2 callers of this method where one actually only wanted a `PackageIdent` (but had to call with `.method().ident.into()`) and the other didn't care about the returning type, only the `Result` part of it. So only one call site needed to be updated--and it involved deleting code there to boot 😄